File: Examples/Python/Examples/RemoveParticularDocumentProperty.py
import asposewordscloud
from asposewordscloud.WordsApi import WordsApi
from asposewordscloud.WordsApi import ApiException
from asposewordscloud.models import ProtectionRequest

import asposestoragecloud
from asposestoragecloud.StorageApi import StorageApi
from asposestoragecloud.StorageApi import ResponseMessage


apiKey = "XXXXX" #sepcify App Key
appSid = "XXXXX" #sepcify App SID
apiServer = "http://api.aspose.com/v1.1"
data_folder = "../../data/"

#Instantiate Aspose Storage API SDK
storage_apiClient = asposestoragecloud.ApiClient.ApiClient(apiKey, appSid, True)
storageApi = StorageApi(storage_apiClient)
#Instantiate Aspose Words API SDK
api_client = asposewordscloud.ApiClient.ApiClient(apiKey, appSid, True)
wordsApi = WordsApi(api_client)

#set input file name
filename = "SampleWordDocument.docx"
propertyName = "AsposeAuthor"

#upload file to aspose cloud storage
storageApi.PutCreate(Path=filename, file=data_folder + filename)

try:
    #invoke Aspose.Words Cloud SDK API to delete document property by given name from a word document
    response = wordsApi.DeleteDocumentProperty(name=filename, propertyName=propertyName)

    if response.Status == "OK":
        print "The document property has been removed successfully"

except ApiException as ex:
            print "ApiException:"
            print "Code:" + str(ex.code)
            print "Message:" + ex.message

File: Examples/Node.js/Document-Properties/RemoveParticularDocumentProperty.js
const { WordsApi, DeleteDocumentPropertyRequest} = require("asposewordscloud");
wordsApi = new WordsApi('78946fb4-3bd4-4d3e-b309-f9e2ff9ac6f9', 'b125f13bf6b76ed81ee990142d841195');

// Set input file name
var fileName = "SampleWordDocument.docx";

request = new DeleteDocumentPropertyRequest();
request.name = fileName;
request.propertyName = "testProp";

wordsApi.deleteDocumentProperty(request).then((result) => {
    console.log(result);
}).catch(function(err) {
    console.log(err);
});

